src/main.ts

```diff
@@ -249,9 +275,17 @@ export default class AdvancedURI extends Plugin {
                         .setIcon("link")
                         .setSection("info")
                         .onClick((_) =>
-                            this.handlers.handleCopyFileURI(true, file)
+                            this.handlers.handleCopyFileURI(true, false, file)
                         );
                 });
+                menu.addItem((item)=>{
```

Comment:
I feel like the file menu is already pretty full. I would rather keep it concise and therefore not add this item.

Reply:
I think need both version, because remove old behaviour will break current scenarios.

And it is may be not comfort as option in settings, because somebody can need both types of link and he will deside about link type at copy moment. It depends from target link (as simple link or paste to another markdown file).

Reply:
Sorry, but I can't follow your comment. Just keep the old menu item with the old function to just copy the uri. The added commands should be used as a way to copy the whole markdown link.

Reply:
Can I add option to settings for show/hide markdown item (hidden by default)?